### PR TITLE
Injecting 3scale credentials to container

### PR DESCRIPTION
**Pull request by GeetikaBatra**
This commit injects 'three_scale_api_access_key',
'three_scale_api_account' and 'three_scale_api_host_url'
from secret keys into the 3scale container.

### CODE REVIEW

**Comment by GeetikaBatra** [comment]
@msrb and @arunkumars08. Please review. Thanks in advance.

File: openshift/template.yaml [hunk 1]
```diff
@@ -27,7 +27,22 @@ objects:
             value: "${GUNICORN_CMD_ARGS}"
           image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
           name: f8a-3scale-admin-gateway
-          ports:
+          - name: THREE_SCALE_API_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: 3scale
+                key: three_scale_api_access_key
+          - name: THREE_SCALE_API_HOST_URL
```
Comment (msrb) [inline openshift/template.yaml]: This is not a secret, we can get the value from config map:

```
          - name: THREE_SCALE_API_ACCESS_KEY
            valueFrom:
              configMapKeyRef:
                name: bayesian-config
                key: three-scale-api-host-url
```

Comment (msrb) [reply openshift/template.yaml]: This change requires https://github.com/fabric8-analytics/fabric8-analytics-common/pull/360.

Comment (GeetikaBatra) [reply openshift/template.yaml]: @msrb, I have removed it. :)

File: openshift/template.yaml [hunk 2]
```diff
@@ -27,7 +27,17 @@ objects:
             value: "${GUNICORN_CMD_ARGS}"
           image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
           name: f8a-3scale-admin-gateway
-          ports:
+          - name: THREE_SCALE_API_ACCESS_KEY
```
Comment (msrb) [inline openshift/template.yaml]: Now I noticed that this should actually go to the `env` section of the template, starting on line 25: https://github.com/fabric8-analytics/3scale-admin-gateway/pull/9/files#diff-8c255b087ddc69d83b81d4fb3a18c9cbL25

I'll also need env. variable for `THREE_SCALE_API_HOST_URL`, which can be taken from the config map:
```
          - name: THREE_SCALE_API_HOST_URL
            valueFrom:
              configMapKeyRef:
                name: bayesian-config
                key: three-scale-api-host-url
```

**Comment by msrb** [comment]
Fixed indentation and added `THREE_SCALE_API_HOST_URL` env. var.

**Comment by msrb** [comment]
Note this will fail now as secrets are missing on staging.

**Comment by GeetikaBatra** [comment]
@msrb , Thanks for merging.
I have filed an issue on housekeeping/issues to add secrets on staging. Is there something I can do to escalate this process.

**Pull request merged**